js/bitget3.js

```diff
@@ -0,0 +1,2899 @@
+            // request['direct'] = 'next'; // or 'prev'
+            request['method'] = 'openOrders';
+            if (limit === undefined) {
+                request['size'] = limit; // default 100, max 1000
+            }
+        } else if (type === 'swap') {
+            method = 'swapGetOrderOrders';
+            request['status'] = '2'; // 0 Failed, 1 Partially Filled, 2 Fully Filled 3 = Open + Partially Filled, 4 Canceling
+            request['from'] = '1';
+            request['to'] = '1';
+            if (limit === undefined) {
+                request['limit'] = 100; // default 100, max 100
+            }
+        }
+        const query = this.omit (params, 'type');
+        const response = await this[method] (this.extend (request, query));
+        //
+        //  spot
+        //
+        //
+        //     {
+        //         "status":"ok",
+        //         "ts":1595875165865,
+        //         "data":[
+        //             {
+        //                 "account_id":"7420922606",
+        //                 "amount":"0.1000000000000000",
+        //                 "canceled_at":"1595872129618",
+        //                 "created_at":"1595872089525",
+        //                 "filled_amount":"0.000000000000",
+        //                 "filled_cash_amount":"0.000000000000",
+        //                 "filled_fees":"0.000000000000",
+        //                 "finished_at":"1595872129618",
+        //                 "id":"671701716584665088",
+        //                 "price":"150.000000000000",
+        //                 "source":"接口",
+        //                 "state":"canceled",
+        //                 "symbol":"eth_usdt",
+        //                 "type":"buy-limit"
+        //             }
+        //         ]
+        //     }
+        //
+        // swap
+        //
+        //     [
+        //         {
+        //             "symbol":"cmt_ethusdt",
+        //             "size":"1",
+        //             "timestamp":"1595885546770",
+        //             "client_oid":"f3aa81d6-9a4c-4eab-bebe-ebc19da21cf2",
+        //             "createTime":"1595885521200",
+        //             "filled_qty":"0",
+        //             "fee":"0.00000000",
+        //             "order_id":"671758053112020913",
+        //             "price":"150.00",
+        //             "price_avg":"0.00",
+        //             "status":"0",
+        //             "type":"1",
+        //             "order_type":"0",
+        //             "totalProfits":null
+        //         }
+        //     ]
+        //
+        let data = response;
+        if (!Array.isArray (response)) {
+            data = this.safeValue (response, 'data', []);
+        }
+        return this.parseOrders (data, market, undefined, limit);
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        if (code === undefined) {
```

Comment:
Not working? 😕 
![image](https://user-images.githubusercontent.com/43336371/152381144-b422d291-7fd5-4647-83ce-630273609199.png)

Reply:
Fixed in PR #11907